installers/win-installer/includes/data.nsh

```diff
@@ -1,74 +1,98 @@
-SubSection "35e"
-	Section "12 to Midnight"
+SubSection "sidewinder"
+	Section "Dog House Rules"
 	SectionIn 1 2
-	SetOutPath "$INSTDIR\${APPDIR}\data\35e\12_to_midnight"
-	File /r "${SrcDir}\PCGen_${SIMPVER}_opt\data\35e\12_to_midnight\*.*"
+	SetOutPath "$INSTDIR\${APPDIR}\data\sidewinder\doghouse_rules"
+	File /r "${SrcDir}\PCGen_${SIMPVER}_opt\data\sidewinder\doghouse_rules\*.*"
 	SectionEnd
 
-	Section "Alderac Entertainment Group"
+SubSectionEnd
+
+SubSection "darwins_world_2"
+	Section "RPG Objects"
 	SectionIn 1 2
-	SetOutPath "$INSTDIR\${APPDIR}\data\35e\alderac_entertainment_group"
-	File /r "${SrcDir}\PCGen_${SIMPVER}_opt\data\35e\alderac_entertainment_group\*.*"
+	SetOutPath "$INSTDIR\${APPDIR}\data\darwins_world_2\rpg_objects"
+	File /r "${SrcDir}\PCGen_${SIMPVER}_opt\data\darwins_world_2\rpg_objects\*.*"
 	SectionEnd
 
-	Section "Alea Publishing Group"
+SubSectionEnd
+
+SubSection "spycraft"
+	Section "Crafty Games"
 	SectionIn 1 2
-	SetOutPath "$INSTDIR\${APPDIR}\data\35e\alea_publishing_group"
-	File /r "${SrcDir}\PCGen_${SIMPVER}_opt\data\35e\alea_publishing_group\*.*"
+	SetOutPath "$INSTDIR\${APPDIR}\data\spycraft\crafty_games"
+	File /r "${SrcDir}\PCGen_${SIMPVER}_opt\data\spycraft\crafty_games\*.*"
 	SectionEnd
 
-	Section "Bad Axe Games"
+SubSectionEnd
+
+SubSection "_images"
```

Comment:
Odd, I included the files cause your requested them, but that must be an autogenerated deal then. Bah...

@cpmeister or @thpr can you review the build portion and push this in if you're cool with the changes.